presto-docs/src/main/sphinx/spill.rst

```diff
@@ -0,0 +1,8 @@
+****************
+Spilling to disk
+****************
+
+.. toctree::
+   :maxdepth: 1
+
+   spill/overview
```

Comment:
We already have brief explanation of spill in `admin` section where spill properties are described (see `beta.spill-enabled` option). Here my goal was to describe the mechanism in more detail.
Also, we recommend enabling spill on per query basis, so the description is not related only to cluster configuration as most of the `admin` section.
That said, I agree with you that spill is not a topic for a top-level section. How about putting it inside `overview` that is not added to main docs? Later we could have another top-level section for spill and optimizer docs.